ramls/move.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Ids holder for moving items between holdings",
+  "type": "object",
+  "properties": {
+    "to": {
+      "description": "Id of the instance to which the holdings are moved, or id of the holding to which the items are moved.",
```

Comment:
How it is going to distinguish whether `to` is an instance ID or a holding ID? Will there be two separate APIs which will have the same schema?

If the above, I'd prefer to have two separate schemes as well:

* Move holdings 
```java
{
  "toInstanceId": required, uuid,
  "holdingsRecordIds": required, array
}
```

* move items:
```java
  "toHoldingsRecordId": required, uuid,
  "itemIds": required, array
}
```

What do you think?

Reply:
Hi @bohdan-suprun ,
we planned to use the same schema for both separate APIs for items and holdings moving. Actually, we don't need to introduce separate schemas for holdings and items because we are going to delegate all processing logic to corresponding API. Even if we send non-existent IDs for some items/holdings we just return the response with list of IDs for wich we could not update items/holdings.

Reply:
It is simple to introduce separate schemas for items/holdings moving. So let me please know if you see reason to make this. Thanks!

Reply:
Thanks, @khandramai. Leaving it with you to decide.

Reply:
> It is simple to introduce separate schemas for items/holdings moving. So let me please know if you see reason to make this.

Yes, I think it will make API clearer for consumers. And less error prone, in terms of specifying wrong UUID for a property.

Reply:
@bohdan-suprun, thanks. It makes sense for me.
@siarhei-charniak could you please implement this proposal?